api/lists.go

```diff
@@ -130,7 +130,7 @@ func (c *ListsController) GetPercentile(ctx *app.GetPercentileListsContext) erro
 	if !ok {
 		return ctx.NotFound(goa.ErrNotFound("not_found", "list_id", ctx.ListID))
 	}
-	percentile := float64(50.0)
+	var percentile float64
```

Comment:
Maybe something like this, now that we are touching the code
```
percentile, err2 := strconv.ParseFloat(ctx.FromTop, 64); 
if err2 != nil {
		return ctx.BadRequest(goa.InvalidParamTypeError("from_top", ctx.FromTop, "number"))	
}
```

Reply:
Didn't seem to be any reason it was named `err2`, so I changed that to `err` while I was cleaning up